### ISSUE TITLE
"An exception has occured" while scanning library

### ISSUE THREAD

**Issue by nodiscc** [quodlibet/quodlibet#2442]
My attempts to scan my library fail after a few seconds. I narrowed the problem to this specific file:

```bash
$ strace -e file -p 25731 2>&1 | grep /path/to/music
[...]
lstat("/path/to/music/Saul Williams - The Inevitable Rise and Liberation of NiggyTardust", {st_mode=S_IFDIR|S_ISGID|0770, st_size=4096, ...}) = 0
lstat("/path/to/music/Saul Williams - The Inevitable Rise and Liberation of NiggyTardust/07 DNA.mp3", {st_mode=S_IFREG|0660, st_size=5965212, ...}) = 0
open("/path/to/music/Saul Williams - The Inevitable Rise and Liberation of NiggyTardust/07 DNA.mp3", O_RDONLY) = 28

[scan stops, "An exception has occured" dialog opens]
```

The file can be found **[here](https://framadrop.org/r/FpRKZw9FgW#3/+Viq6soZAnfeQ3qv0oins6gZMK9V0nY2GID/JUQTU=)**

Dump:

```

=== SYSTEM INFORMATION:
Quod Libet 3.7.1
Mutagen 1.36
Python 2.7.13 (default, Jan 19 2017, 14:48:08) 
[GCC 6.3.0 20170118] linux2
Platform Linux-4.9.0-3-amd64-x86_64-with-debian-9.0
=== STACK TRACE

Traceback (most recent call last):

  File "/usr/lib/python2.7/dist-packages/quodlibet/util/copool.py", line 25, in wrap
    for value in func(*args, **kwargs):

  File "/usr/lib/python2.7/dist-packages/quodlibet/library/libraries.py", line 690, in rebuild
    for value in self.scan(paths, exclude, cofuncid):

  File "/usr/lib/python2.7/dist-packages/quodlibet/library/libraries.py", line 740, in scan
    item = self.add_filename(fullfilename, False)

  File "/usr/lib/python2.7/dist-packages/quodlibet/library/libraries.py", line 840, in add_filename
    song = MusicFile(filename)

  File "/usr/lib/python2.7/dist-packages/quodlibet/formats/_misc.py", line 118, in MusicFile
    return loader(filename)

  File "/usr/lib/python2.7/dist-packages/quodlibet/formats/_id3.py", line 193, in __init__
    text = self.__distrust_latin1(text, frame.encoding)

  File "/usr/lib/python2.7/dist-packages/quodlibet/formats/_id3.py", line 258, in __distrust_latin1
    text = text.encode('iso-8859-1')

UnicodeEncodeError: 'latin-1' codec can't encode character u'\u2028' in position 92: ordinal not in range(256)

=== LOG:
[...]
D: 42.936: SongFileLibrary.add(<libraries.SongFileLibrary object at 0x7fd50a59a0f0 (quodlibet+library+libraries+SongFileLibrary at 0x5651ce028ee0)>): Adding 1 items.
D: 43.530: qltk.completion.__update_song: Updating tag model for 1 songs
D: 43.530: qltk.completion.__update_song: Done updating tag model for 1 songs
D: 43.937: SongFileLibrary.add(<libraries.SongFileLibrary object at 0x7fd50a59a0f0 (quodlibet+library+libraries+SongFileLibrary at 0x5651ce028ee0)>): Adding 11 items.
D: 44.748: qltk.completion.__update_song: Updating tag model for 11 songs
D: 44.748: qltk.completion.__update_song: Done updating tag model for 11 songs
D: 44.940: SongFileLibrary.add(<libraries.SongFileLibrary object at 0x7fd50a59a0f0 (quodlibet+library+libraries+SongFileLibrary at 0x5651ce028ee0)>): Adding 9 items.
D: 45.700: qltk.completion.__update_song: Updating tag model for 9 songs
D: 45.700: qltk.completion.__update_song: Done updating tag model for 9 songs
```

**Comment by nodiscc** [comment]
Clearing the id3v2 `comment` tag for that file makes the problem vanish. The previous value was:

```
1\\The Inevitable Rise and Liberation of NiggyTardust! Written and performed by Saul Williams with Trent Reznor, CX Kidtronik, Thavius Beck and more. Produced by Trent Reznor Mixed By Alan Moulder Mastered by Brian "Big Bass" Gardner Engineering / programmin
```

Had to use puddletag to remove this tag since Ex Falso would also crash on the file.

**Comment by lazka** [comment]
Thanks. That was fixed in 3.9.1

**Comment by lazka** [comment]
(or updating to mutagen 1.38 should fix it as well)
